### PR TITLE
Clarify documentation about special case shields

### PR DESCRIPTION
**Pull request by claysmalley**
~Removes a line of documentation describing "special code" that does not exist.~

Stacked refs in Italian motorways are a thing, but the current implementation displays the `ref=*` verbatim on one line (#366).

### CODE REVIEW

**Comment by zekefarwell** [comment]
I don't think the intent with this chunk of text was to imply that the Italian branch motorway refs were currently supported, but just to document them as a special case that would require special handling.  Perhaps rather than "This requires special code ...", better phrasing would be "These are currently not supported.  Potential future support will require special code ...".  Another alternative to simply removing this from the documenation would be splitting the **Special Cases** list into two lists: **Supported** and **Not Supported**.

**Comment by claysmalley** [comment]
Rearranged the special cases section and added the Italy example back in, with clarification that it is not implemented yet.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -267,10 +267,15 @@ In all cases, banner text should be no more than **4** characters in length.
...
+- Shields for route networks where each individual route is identified by a color, rather than a number or letter. Such cases include:
+  - **Allegheny County, PA Belt Routes**. Shields for this system use colors, with a colored circle and the words "<COLOR> BELT". These shields are drawn as squares with colored circles, with the `ref` values correctly corresponding to the text on the shield. Because of the common design (white shield with colored circle), these shields are properly part of a common route network. Special code is needed to convert the textual ref values to the colors displayed in the shield.
+- Shields with a stacked ref configuration, with `/` separating the two lines of text in the `ref` value. Currently, these `ref` values are displayed verbatim on one line, and the code necessary for stacked ref rendering has not been written yet (#366). Such cases include:
```
Comment (ZeLonewolf) [inline CONTRIBUTING.md:279]: I don't think #366 will link in this way from a markdown file.

Comment (claysmalley) [reply CONTRIBUTING.md]: Fixed in merge commit, grumble grumble

**Review by ZeLonewolf — APPROVED** [review]
Looks good 👍

**Pull request merged**